{
  "parameter(s)": [
    94
  ],
  "test_list_of_primes": [
    "failed",
    "ValueError: 100363001 is not in list",
    0.00013
  ]
}